index.fr.md

```diff
@@ -0,0 +1,226 @@
+
+<mark><strong>Attention :</strong> prêtez attention à toutes les données. <strong>Évitez de supposer que les données d'une personne présentant une certaine forme de handicap s'appliquent à toutes les personnes handicapées. </strong></mark>
+
+Une personne handicapée ne sait pas forcément comment d'autres personnes présentant le même type de handicap interagissent avec le Web. Elle pourrait ne pas en savoir assez sur les autres handicaps pour fournir une aide valable pour d'autres questions d'accessibilité. Il est préférable de rassembler des données de plusieurs types d'utilisateurs.
+
+Note : en plus de détecter des problèmes d'accessibilité, l'évaluation menée avec des utilisateurs handicapés révèlent des problèmes généraux d'utilisation qui touchent tous les utilisateurs, y compris ceux qui ne se trouvent *pas* en situation de handicap.
+
+## Analyse des problèmes d'accessibilité {#analyz}
+
+L'accessibilité Web dépend de l'association [des différents composants du développement Web avec l'interaction](/fundamentals/components/), dont les navigateurs Web, les technologies d'assistance, et le contenu Web.
+
+Les problèmes d'accessibilité peuvent être causés par un ou plusieurs composants. Par exemple, si un utilisateur qui ne peut pas utiliser une souris n'arrive pas à accéder aux fonctionnalités clavier des technologies d'assistance, il peut y avoir plusieurs explications :
+
+-   le développeur n'a pas balisé/codé correctement la page Web, ou
+-   le navigateur ou le lecteur multimédia ne lit pas correctement le code, ou
+-   la technologie d'assistance ne lit pas correctement le code, ou
+-   l'utilisateur ne sait pas comment utiliser le navigateur ou le lecteur multimédia ou les fonctionnalités accessibles au clavier des technologies d'assistance, ou
+-   la page n'est pas correment conçue et il s'agit d'un problème général d'utilisation pour tous les utilisateurs, y compris ceux qui ne se trouvent *pas* en situation de handicap.
+s
+## Associez l'évaluation des utilisateurs aux standards {#stdstoo}
+
+Inclure les utilisateurs handicapés dans l'évaluation présente de nombreux avantages. Ceci dit, cela ne suffit pas à déterminer l'accessibilité d'un site Web. Associez l'inclusion des utilisateurs à l'[évaluation de la conformité aux WCAG](/test-evaluate/conformance/) afin de garantir l'accessibilité pour différents types de handicaps et de situations.
+
+## Tirez des conclusions et créez un rapport {#drawing}
+
+Faites attention si vous tirez des conclusions d'évaluations ou d'études limitées. Les résultats obtenus avec deux personnes handicapées ne peuvent être généralisés et appliqués à toutes les personnes dans la même situation de handicap ou présentant d'autres handicaps. Retrouvez plus d'informations dans la section [Attention](#caution) ci-dessous.
```

Comment:
- "deux personnes en situation de handicap"
- The caution section is "above" (ci-dessus) not below (ci-dessous).
- There is no anchor link for the "caution" section. But the problem lies in the original version : I will suggest an edit.